ci/lint/06_script.sh

```diff
@@ -31,6 +31,10 @@ if [ "$CIRRUS_REPO_FULL_NAME" = "bitcoin/bitcoin" ] && [ "$CIRRUS_PR" = "" ] ; t
     git log HEAD~10 -1 --format='%H' > ./contrib/verify-commits/trusted-sha512-root-commit
     git log HEAD~10 -1 --format='%H' > ./contrib/verify-commits/trusted-git-root
     mapfile -t KEYS < contrib/verify-commits/trusted-keys
+    if [ "$CI" = "true" ]; then
```

Comment:
maybe remove this check, since there is already the `[ "$CIRRUS_REPO_FULL_NAME" = "bitcoin/bitcoin" ]` check?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26236#issuecomment-1266546245).